src/components/search/detailed/styles.js

```diff
@@ -12,7 +12,7 @@ export default (theme) => ({
         [theme.breakpoints.up("md")]: {
             minWidth: "50%",
         },
-        [theme.breakpoints.down("md")]: {
+        [theme.breakpoints.down("lg")]: {
```

Comment:
Pretty sure this was previously a bug, but reverting these code-mod changes will fix the bug in this PR.